scripts/results_generation/results_gmcc_metric.py

```diff
@@ -0,0 +1,254 @@
+                                     "door_closing/door_closing_repro.json")
+    gmcc_value_dc = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Door opening score
+    regression_path = os.path.join(data_dir, "door_opening/tcp_regression.npy")
+    reproduction_path = os.path.join(data_dir,
+                                     "door_opening/door_opening_repro.json")
+    gmcc_value_do = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Rail cleaning score
+    regression_path = os.path.join(data_dir, "rail_cleaning/tcp_regression.npy")
+    reproduction_path = os.path.join(data_dir,
+                                     "rail_cleaning/rail_cleaning_repro.json")
+    gmcc_value_rc = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Drill task score
+    regression_path = os.path.join(data_dir, "drill/regression/demonstration_1.json")
+    reproduction_path = os.path.join(data_dir, "drill/drill_repro/demonstration_1.json")
+    gmcc_value_drill = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Drill far task score
+    regression_path = os.path.join(data_dir, "drill/regression/demonstration_1.json")
+    reproduction_path = os.path.join(data_dir, "drill/drill_far/demonstration_1.json")
+    gmcc_value_dr_f = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Timo avoid task score
+    regression_path = os.path.join(data_dir,
+                                   "Timo_avoid/regression/demonstration_1.json")
+    reproduction_path = os.path.join(data_dir,
+                                     "Timo_avoid/repro/demonstration_1.json")
+    gmcc_value_ta = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Timo avoid far task score
+    regression_path = os.path.join(data_dir,
+                                   "Timo_avoid/regression/demonstration_1.json")
+    reproduction_path = os.path.join(data_dir,
+                                     "Timo_avoid/repro_far/demonstration_1.json")
+    gmcc_value_ta_f = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Timo collab task score
+    regression_path = os.path.join(data_dir,
+                                   "Timo_collab/regression/demonstration_1.json")
+    reproduction_path = os.path.join(data_dir,
+                                     "Timo_collab/repro/demonstration_1.json")
+    gmcc_value_tc = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Timo collab far task score
+    regression_path = os.path.join(data_dir,
+                                   "Timo_collab/regression/demonstration_1.json")
+    reproduction_path = os.path.join(data_dir,
+                                     "Timo_collab/repro_far/demonstration_1.json")
+    gmcc_value_tc_f = get_gmcc_score(regression_path, reproduction_path, "tcp")
+    # Timo difficult task score
+    regression_path = os.path.join(data_dir,
+                                   "Timo_difficult/regression/demonstration_1.json")
```

Comment:
THere is a typo in the folder name that is named `TImo_difficult` instead of `Timo_difficult` so it crashes on my linux machine :P